pkg/virt-controller/watch/vmi_test.go

```diff
@@ -3284,8 +3284,11 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 					appendNetworkToVMI(
 						api.NewMinimalVMI(vmName), firstVMNetwork, firstVMInterface),
 					secondVMNetwork, secondVMInterface)
-				pod = NewPodForVirtualMachine(vmi, k8sv1.PodRunning)
-				Expect(pod.Annotations).To(HaveKey(networkv1.NetworkAttachmentAnnot))
```

Comment:
No need to remove this one.

Reply:
Note that this test is context of hotplug interface tests, and NewPodForVirtualMachineWithMultusAnnotations() set multus annotation on the pod.